src/main/scala/io/qbeast/core/model/QDataType.scala

```diff
@@ -72,31 +72,49 @@ trait OrderedDataType extends QDataType {
   @JsonIgnore
   val ordering: Numeric[Any]
 
+  val defaultQuantiles: IndexedSeq[Any]
+
 }
 
 object DoubleDataType extends OrderedDataType {
   override def name: String = "DoubleDataType"
   override val ordering: Numeric[Any] = implicitly[Numeric[Double]].asInstanceOf[Numeric[Any]]
+
+  override val defaultQuantiles: IndexedSeq[Any] =
```

Comment:
Not sure if we should support a `defaultQuantiles` value or should we throw an error when no quantiles are specified. What do you think @cugni ?

Reply:
We should raise an exception if not provided, IMHO.